website/project/metadata/asist-hypothesis-capability-registration.json

```diff
@@ -0,0 +1,32 @@
+{
+   "name":"ASIST Hypothesis/Capability Registration",
+   "version":1,
+   "config":{
+      "hasFiles":true
+   },
+   "description":"This template was created by researchers from the qualitative community for preregistration of qualitative research",
```

Comment:
According to [the ticket](https://openscience.atlassian.net/browse/ENG-2306), this should be, "This preregistration template is part of the DARPA Artificial Social Intelligence for Successful Teams (ASIST) program. This template is intended to formally register performer hypothesis and AI capability preregistrations throughout the ASIST program."